lib/membrane_opus.ex

```diff
@@ -1,12 +1,12 @@
 defmodule Membrane.Opus do
   @moduledoc """
-  This module implements struct describing an Opus-encoded audio stream.
+  This module implements struct describing an Opus-encoded audio packet.
```

Comment:
The format always describes a stream, not a single packet. Packet-specific information should be passed in the buffer's metadata.